docs/blog/2022-02-02-Changes_to_Slurm_srun_for_interactive_jobs.md

```diff
@@ -0,0 +1,15 @@
+# Changes to Slurm "srun" for Interactive Jobs
+
+The Slurm job scheduler was upgraded during the recent system time on January 10, 2022 . 
```

Comment:
Looks good except for extra space before the period here. These entries also need to be added to the mkdocs.yml navigation section.

Reply:
Done!